tests/integration/python/test_python.py

```diff
@@ -20,16 +20,27 @@ def models(self):
     def project_config(self):
         return {
             "config-version": 2,
-            "vars": {"http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH")},
+            "vars": {
+                "http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH"),
+                "location_root": "",
+            },
         }
 
     def python_exc(self):
-        self.run_dbt(["run"])
+        self.run_dbt(["run", "-s", "basic"])
```

Comment:
Assuming we can handle a default null `location_root`, this line feels like the only necessary change to `test_python.py` in this pull request.